esmvaltool/recipes/recipe_Ocean_scarlar_fields.yml

```diff
@@ -0,0 +1,60 @@
+###############################################################################
+# recipe_scarlar_fields.yml
+---
+documentation:
+  description: |
+    Recipe to demonstrate several simple plots based on the scalar fields.
+    These fluxes are saved as MFO files in CMIP5.
+    Please use this file as a template for adding additional
+    fields into the ocean. This work based on the BGC-val toolkit GMD-2018-103.
+    Written by Lee de Mora, Plymouth Marine Laboratory
+
+  authors:
+    - ledm
```

Comment:
Please make sure the authors are defined in the file esmvaltool/config-references.yml